### PR TITLE
Add python test for setting RO attribute

### PR DESCRIPTION
**Pull request by kcudnik**

### CODE REVIEW

**Comment by lguohan** [comment]
the test looks ok, I think you need first merge the swss-common, wait till the swss-common is compiled, and then restart test, then vs test will pass.

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
retest this please

**Comment by lguohan** [comment]
test failed, @kcudnik, can you check?

**Comment by kcudnik** [comment]
yes, correct

File: tests/test_setro.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    tbl = swsscommon.Table(db, "ASIC_STATE:SAI_OBJECT_TYPE_SWITCH")
+
+    keys = tbl.getKeys()
+
+    assert len(keys) == 1
+
+    swVid = keys[0]
+ 
+    rc = swsscommon.RedisClient(db)
```
Comment (lguohan) [inline tests/test_setro.py]: no more needed.

Comment (kcudnik) [reply tests/test_setro.py]: rremoved

**Pull request merged**